en/simulation/multi-vehicle-simulation.md

```diff
@@ -39,88 +38,72 @@ To test an example setup, follow the below steps,
 
 For each simulated vehicle, the following is required,
 
-* **Gazebo model**: This is defined as `xacro` file in `Firmware/Tools/sitl_gazebo/models/rotors_description/urdf/<model>_base.xacro` see [here](https://github.com/PX4/sitl_gazebo/tree/02060a86652b736ca7dd945a524a8bf84eaf5a05/models/rotors_description/urdf). Currently, the model `xacro` file is assumed to end with base.xacro. This model should have an argument called  `mavlink_udp_port` which defines the UDP port on which gazebo will communicate with PX4 node. The model's `xacro` file will be used to generate an `sdf` model that contains UDP port that you select. To define the UDP port, set the `mavlink_udp_port` in the launch file, see [here](https://github.com/PX4/Firmware/blob/master/launch/multi_uav_mavros_sitl.launch#L48).  
+* **Gazebo model**: This is defined as `xacro` file in `Firmware/Tools/sitl_gazebo/models/rotors_description/urdf/<model>_base.xacro` see [here](https://github.com/PX4/sitl_gazebo/tree/02060a86652b736ca7dd945a524a8bf84eaf5a05/models/rotors_description/urdf). Currently, the model `xacro` file is assumed to end with base.xacro. This model should have an argument called  `mavlink_udp_port` which defines the UDP port on which gazebo will communicate with PX4 node. The model's `xacro` file will be used to generate an `urdf` model that contains UDP port that you select. To define the UDP port, set the `mavlink_udp_port` in the launch file for each vehicle, see [here](https://github.com/PX4/Firmware/blob/4d0964385b84dc91189f377aafb039d10850e5d6/launch/multi_uav_mavros_sitl.launch#L37) as an example.  
   **NOTE: If you are using the same vehicle model, you don't need  a separate **`xacro`** file for each vehicle. The same **`xacro`** file is adequate.                
```

Comment:
Can you change this from:
```
**NOTE: If you are using the same vehicle model, you don't need  a separate **`xacro`** file for each vehicle. The same **`xacro`** file is adequate.    
```
TO
```
> **Note** If you are using the same vehicle model, you can use the same **xacro** file for each vehicle.
``` 
where the `>` is at the beginning of the line.